build/shade/_k-standard-goals.shade

```diff
@@ -230,7 +239,7 @@ default SAMPLES_PROJECT_GLOB = "samples/*/project.json"
     var projectFiles = Files.Include(TEST_PROJECT_GLOB);
     foreach (var projectFile in projectFiles)
     {
-        DotnetTest(projectFile, Configuration, " --no-build");
+        DotnetTest(projectFile, Configuration, E("KOREBUILD_DOTNET_TEST_OPTIONS") + " --no-build");
```

Comment:
Can we remove the default value from _dotnet-test.shade? 

https://github.com/aspnet/KoreBuild/blob/dev/build/shade/_dotnet-test.shade#L19